### PR TITLE
Add new blog post by Charles Goode

### PR DESCRIPTION
**Pull request by jdorfman**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206827268857463

### CODE REVIEW

File: content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: "Crossing the Finish Line Faster: Leveraging Cody to Implement GPS Data Logging "
```
Comment (kukicado) [inline content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md:2]:
```suggestion
title: "Crossing the Finish Line Faster: Leveraging Cody to Implement GPS Data Logging"
```

File: content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+authors:
+  - name: Charles Goode
+  
+publishDate: 2024-03-12T10:00-07:00
+description: "Charles Goode, an electrical engineering student at Kennesaw State University, shares his experience using the Cody AI assistant to rapidly implement a GPS data logging system for the university's Formula SAE racing team, Kennesaw Motorsports."
```
Comment (kukicado) [inline content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md:7]:
```suggestion
description: "Charles Goode, an electrical engineering student at Kennesaw State University, shares his experience using Cody to rapidly implement a GPS data logging system for the university's Formula SAE racing team, Kennesaw Motorsports."
```

File: content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+slug: "leveraging-cody-to-implement-gps-data-logging"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/og-leveraging-cody-to-implement-gps-data-logging.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/og-leveraging-cody-to-implement-gps-data-logging.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+Hello, my name is Charles Goode and I’m an electrical engineering major at Kennesaw State University. I’m here to tell you about Kennesaw Motorsports and to walk you through how I completed my first project on the team with the help of Cody. 
+
```
Comment (kukicado) [inline content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md:18]:
```suggestion
```

File: content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+tags: [blog, guest-post]
+slug: "leveraging-cody-to-implement-gps-data-logging"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/og-leveraging-cody-to-implement-gps-data-logging.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/og-leveraging-cody-to-implement-gps-data-logging.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+Hello, my name is Charles Goode and I’m an electrical engineering major at Kennesaw State University. I’m here to tell you about Kennesaw Motorsports and to walk you through how I completed my first project on the team with the help of Cody.
```
Comment (kukicado) [inline content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md:17]:
```suggestion
Hello, my name is Charles Goode and I’m an electrical engineering major at Kennesaw State University. I’m here to tell you about Kennesaw Motorsports and to walk you through how I completed my first project on the team with the help of [Cody](https://sourcegraph.com/cody). 
```

File: content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+ 
+All the data is formatted correctly, but we need some real data to plot this correctly. The next time we tested one of our cars, I collected some real data from the CSV file. Now that we have the data needed, we must visualize it. There are a few ways to do this, but I picked MATLAB. I cannot do this inside my IDE, but Cody can still be helpful. 
+ 
+![](https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/image_007.png)
+
+I asked Cody for a MATLAB script, and it was quickly delivered. I went to MATLAB, pasted in the script, and it ran perfectly. Once I input the data from our testing day, it plotted it. Here is the data visualized using the plot function.
+
+![](https://storage.googleapis.com/sourcegraph-assets/blog/leveraging-cody-to-implement-gps-data-logging/image_008.png)
+
+As you can see, Cody was able to help me complete my first project quickly.
```
Comment (kukicado) [inline content/blogposts/2024/leveraging-cody-to-implement-gps-data-logging.md:90]:
```suggestion
As you can see, Cody was able to help me complete my first project quickly. 

```

**Review by kukicado — CHANGES_REQUESTED** [review]
Looks great and an awesome read. Just left a few tiny comments.

**Review by jdorfman — COMMENTED** [review]
👍

**Review by kukicado — APPROVED** [review]
LGTM!

**Pull request merged**